### PR TITLE
faq covering principal cap and individual contributor

### PR DESCRIPTION
**Pull request by wheresrhys**
We've had feedback about individual contributor routes and lack of promotion to principal. Don't apply to any particular competency, or existing level, so created an FAQ section to hold general qualms such as this

### CODE REVIEW

File: site/pages/faq.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# {{page.title}}
+
+## Is there an individual contributor route?
+
+There is no separate individual contributor route to promotion, but the aim is
+to have a single set of competencies that are flexible enough to allow for a
+variety of ways that an engineer can demonstrate seniority.
+
+A key idea here is that the more senior you are as an engineer, the more you
+should be helping those around you to achieve. There are many ways to achieve
```
Comment (rowanmanning) [inline site/pages/faq.html]: Slight jar with repeated "achieve". Suggestion:
```suggestion
should be helping those around you to achieve. There are many ways to do
```

File: site/pages/faq.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+this:
+- Line managing
+- Mentoring
+- Directing the work of a team ("tech leading")
+- Reviewing and driving up the quality in others' technical output
+- Influencing technical communities and decisions across the organisation
+
+Choosing not to pursue line management will mean you need to demonstrate
+that you are having a psitive impact oon others' work in other ways.
```
Comment (rowanmanning) [inline site/pages/faq.html]: typo
```suggestion
that you are having a positive impact on others' work in other ways.
```

File: site/pages/faq.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+in 2018: Principal roles and beyond must have a position open and advertised.
+The promotions board therefore plays no role in deciding who becomes a
+Principal Engineer, so there are no competencies defined at this stage. The
+recruiting manager for each Principal Engineer role defines what the criteria
+for that role will be.
+
+There is no specific cap on the number of Principals; it is governed by open roles
+and decisions about the organisation’s structure. To advance to Principal Engineer
+either a position needs to become vacant, or PTE/TLG need to approve the
+creation of a new Principal Engineer role, which will typically only happen
```
Comment (lc512k) [inline site/pages/faq.html]: > PTE/TLG

Afaik this is the tech director's decision if their budget permits and it's in the interest of the tram (not necessarily due to a massive expansion either). Might be some tgl consultation but I don't think PTE are involved.

Comment (wheresrhys) [reply site/pages/faq.html]: Thanks for the info (I basically made up what was there). I've reworded to correct that, but have left in some examples of when PE roles get created; don't want to leave it at 'new role must be created', as then that begs the question of 'well - when does that happen then?'

**Comment by wheresrhys** [comment]
CI did pass, but hook not reporting 😠 

But I am an admin 😋

**Pull request merged**